### DISCUSSION TITLE
CURLOPT_*_BLOB for alt-svc and cookies?

### DISCUSSION THREAD

**Discussion by Keitagit-kun** [curl/curl#9494] [Ideas]
Is it possible to add CURLOPT_*_BLOB for alt-svc and cookies? It would be nice to have less hdd-access especially in multy-threaded cases...

**Comment by bagder** [comment]
The `BLOB` options have so far primarily been done for options that are read-only. alt-svc and cookies are very much read+write. How would curl write/update/extend to the blob?

**Comment by Keitagit-kun** [comment]
at the beginning I pass to curl pointer to bob/string data. Curl makes internal copy and use this copy as needed (read/write/update/extend). 
In the end (after perform and before cleanup) I can can request pointer to rezulting blob and save it or pass to another curl instanse.
